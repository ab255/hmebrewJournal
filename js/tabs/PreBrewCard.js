import React, {Component} from 'react';
import {
  ScrollView,
  Text,
  View,
} from 'react-native';

class PreBrewCard extends Component {
  constructor(props) {
    super()
  }

  render() {
    return (
      <ScrollView>
        <View>
          <Text>Beer Name</Text>
          <Text>{this.props.beerName}</Text>
        </View>
        <View>
          <Text>Beer Type</Text>
          <Text>{this.props.beerType}</Text>
        </View>
        <View>
          <Text>Ingredients</Text>
          <Text>{this.props.ingredients}</Text>
        </View>
        <View>
          <Text>Water Notes</Text>
          <Text>{this.props.water}</Text>
        </View>
        <View>
          <Text>Post Brew Ingredients</Text>
          <Text>{this.props.postBrewIngredients}</Text>
        </View>
        <View>
          <Text>Total Cost</Text>
          <Text>{this.props.totalCosts}</Text>
        </View>
      </ScrollView>
    )
  }
}

export default PreBrewCard;
