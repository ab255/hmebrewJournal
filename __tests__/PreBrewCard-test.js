import 'react-native';
import React from 'react';
import PreBrewCard from '../js/tabs/PreBrewCard.js';

import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(
    <PreBrewCard />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
