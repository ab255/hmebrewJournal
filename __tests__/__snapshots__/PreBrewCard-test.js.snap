exports[`test renders correctly 1`] = `
<ScrollView
  style={
    Object {
      "padding": 10,
    }
  }>
  <View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#911F27",
          "fontSize": 20,
          "fontWeight": "bold",
          "marginBottom": 7,
          "textDecorationLine": "underline",
        }
      }>
      Beer Name
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#630A10",
          "fontSize": 16,
          "marginBottom": 10,
        }
      } />
  </View>
  <View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#911F27",
          "fontSize": 20,
          "fontWeight": "bold",
          "marginBottom": 7,
          "textDecorationLine": "underline",
        }
      }>
      Beer Type
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#630A10",
          "fontSize": 16,
          "marginBottom": 10,
        }
      } />
  </View>
  <View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#911F27",
          "fontSize": 20,
          "fontWeight": "bold",
          "marginBottom": 7,
          "textDecorationLine": "underline",
        }
      }>
      Ingredients
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#630A10",
          "fontSize": 16,
          "marginBottom": 10,
        }
      } />
  </View>
  <View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#911F27",
          "fontSize": 20,
          "fontWeight": "bold",
          "marginBottom": 7,
          "textDecorationLine": "underline",
        }
      }>
      Water Notes
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#630A10",
          "fontSize": 16,
          "marginBottom": 10,
        }
      } />
  </View>
  <View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#911F27",
          "fontSize": 20,
          "fontWeight": "bold",
          "marginBottom": 7,
          "textDecorationLine": "underline",
        }
      }>
      Post Brew Ingredients
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#630A10",
          "fontSize": 16,
          "marginBottom": 10,
        }
      } />
  </View>
  <View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#911F27",
          "fontSize": 20,
          "fontWeight": "bold",
          "marginBottom": 7,
          "textDecorationLine": "underline",
        }
      }>
      Total Cost
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#630A10",
          "fontSize": 16,
          "marginBottom": 10,
        }
      } />
  </View>
</ScrollView>
`;
